test/ordered_set_test.dart

```diff
@@ -111,23 +111,23 @@ void main() {
 
       test('with identical elements', () {
         final a = OrderedSet<int>();
-        expect(a.addAll([4, 3, 3, 2, 2, 2, 1]), 7);
-        expect(a.length, 7);
-        expect(a.toList().join(), '1222334');
+        expect(a.addAll([4, 3, 3, 2, 2, 2, 1]), 4);
+        expect(a.length, 4);
+        expect(a.toList().join(), '1234');
```

Comment:
I think some of these tests were intended to test the main premise of ordered_set: that multiple elements with the same priority can be added. maybe we should have a new test for that with non-identical but same-priority elements?

Reply:
There are tests for that too, check line 100 for example.

Reply:
And this test actually tests that too since they are all priority 0 right?

Reply:
Added another test for that.